server/server.go

```diff
@@ -767,7 +774,7 @@ func (s *Server) Start() error {
 	go s.ScheduledExecutorService.Run()
 
 	go func() {
-		s.LogStreamingController.Listen()
+		s.ProjectCmdOutputHandler.Handle()
```

Comment:
You can just directly call `projectCmdOutputHandler.Handle()` no need to assign it to server at this point.

Reply:
```suggestion
		projectCmdOutputHandler.Handle()
```